templates/app-settings.js

```diff
@@ -0,0 +1,8 @@
+AppSettings = {
```

Comment:
@roberto mentioned that you need to add a test (the coment is in the pull request). Once you do this I believe we can merge.